sphinx/source/docs/first_steps/first_steps_1.rst

```diff
@@ -69,7 +69,7 @@ Follow these steps to recreate this simple line chart:
         y = [6, 7, 2, 4, 5]
 
 3. Use the :func:`~bokeh.plotting.figure` function to create a new
-   :class:`~bokeh.plotting.figure` object. Pass the following arguments:
+   :class:`~bokeh.plotting.Figure` object. Pass the following arguments:
```

Comment:
Why capital-F Figure? We only ever demonstrate using little-f `figure` everywhere

Reply:
Good point, I keep mixing those up... I have removed any reference to capital-F Figure with the next commit.